src/main/kotlin/no/skatteetaten/aurora/mean/genie/controller/security/WebSecurityConfig.kt

```diff
@@ -1,23 +1,35 @@
 package no.skatteetaten.aurora.mean.genie.controller.security
 
+import javax.servlet.http.HttpServletRequest
+import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.beans.factory.annotation.Value
 import org.springframework.context.annotation.Configuration
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder
 import org.springframework.security.config.annotation.web.builders.HttpSecurity
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter
 import org.springframework.security.config.http.SessionCreationPolicy
+import org.springframework.security.crypto.password.PasswordEncoder
 import org.springframework.security.web.authentication.www.BasicAuthenticationEntryPoint
 import org.springframework.security.web.util.matcher.RequestMatcher
-import javax.servlet.http.HttpServletRequest
 
 @Configuration
 @EnableWebSecurity
 class WebSecurityConfig(
```

Comment:
Er det egentlig behov for dette? Ser ikke ut til at komponenten eksponerer noen endepunkter?

Reply:
Sletter hele fila